Tests/Realm.Tests/Sync/SynchronizedInstanceTests.cs

```diff
@@ -320,6 +320,30 @@ public void EmbeddedObject_WhenAdditiveExplicit_ShouldThrow()
             Assert.Throws<RealmSchemaValidationException>(() => Realm.GetInstance(conf), $"Embedded object {nameof(EmbeddedLevel3)} is unreachable by any link path from top level objects");
         }
 
+        [Test]
+        public void DeleteRealmWorksIfCalledMultipleTimes()
+        {
+            var config = GetFakeConfig();
+            var openRealm = GetRealm(config);
+            openRealm.Dispose();
+            Assert.That(File.Exists(config.DatabasePath));
+            Assert.DoesNotThrow(() => Realm.DeleteRealm(config));
+            Assert.That(File.Exists(config.DatabasePath), Is.False);
+
+            Assert.DoesNotThrow(() => Realm.DeleteRealm(config));
+        }
+
+        [Test]
+        public void DeleteRealmWorksIfFolderDoesntExist()
+        {
+            var config = GetFakeConfig();
+            var dbFolder = Path.GetDirectoryName(config.DatabasePath);
+            var nonExistingRealm = Path.Combine(dbFolder, "idontexist", "my.realm");
+            var newConfig = new RealmConfiguration(nonExistingRealm);
+
+            Assert.DoesNotThrow(() => Realm.DeleteRealm(newConfig));
+        }
```

Comment:
This test doesn't need to be in SynchronizedInstanceTests - it can be rewritten using the local-only API, I just put it in here accidentally 😅

Reply:
Done.